ladybug/src/main/java/nl/nn/ibistesttool/IbisDebuggerAdvice.java

```diff
@@ -169,6 +169,11 @@ public PipeRunResult debugValidatorInputOutputAbort(ProceedingJoinPoint proceedi
 		String messageId = pipeLineSession.getMessageId();
 		message = ibisDebugger.pipeInput(pipeLine, validator, messageId, message);
 		PipeRunResult pipeRunResult = null;
+
+		if(StringUtils.isNotEmpty(messageRoot)) {
+			ibisDebugger.showValue(messageId, "ResponseRoot", messageRoot);
```

Comment:
Het is een beetje 'jumping into conclusions'  om hier het label 'responseRoot' aan mee te geven. Er kan zomaar een situatie ontstaan dat het ook voor requests gebruikt gaat worden. Iets als 'Mandatory Message Root' lijkt me beter.